app/views/users/_form.html.erb

```diff
@@ -0,0 +1,46 @@
+
+<!-- <form action="users_path" method="POST" accept-charset="UTF-8">
+  <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
+  <label for="username">Username:</label>
+  <input type="text" id="username" name="user[username]"><br><br>
+  <label for="email">Email:</label>
+  <input type="email" id="email" name="user[email]"><br><br>
+  <label for="password">Password:</label>
+  <input type="password" id="password" name="user[password]"><br><br>
+  <input type="submit" value="Submit">
+</form> -->
+
+
+<!-- <%= form_tag(users_path) do %>
+  <%= label_tag :username %>
+  <%= text_field_tag :username, params[:username] %>
+  <%= label_tag :email %>
+  <%= text_field_tag :email, params[:email] %>
+  <%= label_tag :password %>
+  <%= password_field_tag :password %>
+  <%= submit_tag "Register" %>
+<% end %> -->
+
+<%= form_with model: user do |form| %>
+  <% flash.each do |type, msg| %>
+    <div>
+      <%= msg %>
+    </div>
+  <% end %>
+
+  <% if @user.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(@user.errors.count, "error") %> prohibited this user from being saved:</h2>
+
+      <ul>
+      <% @user.errors.full_messages.each do |msg| %>
+        <li><%= msg %></li>
+      <% end %>
+      </ul>
+    </div>
+  <% end %>
```

Comment:
- [ ] [OPTIONAL] In general it's not a good practice to have block logic, if/else conditionals and loops, in the views because that makes your app slower. Consider refactoring this logic into a helper method before merging.